ui/src/flux/components/TagValueListItem.tsx

```diff
@@ -65,35 +65,33 @@ class TagValueListItem extends PureComponent<Props, State> {
             </div>
           </CopyToClipboard>
         </div>
-        {this.state.isOpen && (
-          <>
-            {this.isLoading && <LoaderSkeleton />}
-            {!this.isLoading && (
-              <>
-                {!!this.tags.length && (
-                  <div className="flux-schema--filter">
-                    <input
-                      className="form-control input-xs"
-                      placeholder={`Filter within ${value}`}
-                      type="text"
-                      spellCheck={false}
-                      autoComplete="off"
-                      value={searchTerm}
-                      onClick={this.handleInputClick}
-                      onChange={this.onSearch}
-                    />
-                  </div>
-                )}
-                <TagList
-                  db={db}
-                  service={service}
-                  tags={this.tags}
-                  filter={this.filter}
-                />
-              </>
-            )}
-          </>
-        )}
+        <div className={`flux-schema--children ${isOpen ? '' : 'hidden'}`}>
+          {this.isLoading && <LoaderSkeleton />}
+          {!this.isLoading && (
```

Comment:
not related to this changeset, but this `{this.isLoading` and subsequent `{!this.isLoading` pattern is interesting to see -- having mixed feelings about it, like it might be safer or to have a method that generates what is wanted for rendering based on a simple if/then. but this is out of scope for this PR, so just thinking out loud.

Reply:
totally agree—conditional rendering in JSX always feels awkward to me, especially like this.

Reply:
👍